compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUSelectLoweringStrategy.cpp

```diff
@@ -41,24 +29,8 @@ class LLVMGPUSelectLoweringStrategyPass final
       LLVMGPUSelectLoweringStrategyPass>::LLVMGPUSelectLoweringStrategyPassBase;
 
   void getDependentDialects(DialectRegistry &registry) const override {
-    // TODO(qedawkins): Once TransformStrategies is deprecated, drop the
-    // unnecessary dialect registrations.
-    // clang-format off
     registry
-        .insert<IREE::Codegen::IREECodegenDialect,
-                IREE::GPU::IREEGPUDialect,
-                IREE::HAL::HALDialect,
-                IREE::LinalgExt::IREELinalgExtDialect,
-                linalg::LinalgDialect,
-                gpu::GPUDialect,
-                nvgpu::NVGPUDialect,
-                pdl::PDLDialect,
-                pdl_interp::PDLInterpDialect,
-                scf::SCFDialect,
-                tensor::TensorDialect,
-                transform::TransformDialect,
-                vector::VectorDialect>();
-    // clang-format on
```

Comment:
I'm not sure if we should be dropping linalg and linalg_ext here. Doesn't the pass work on linalg dialect?